.github/workflows/conformance.yml

```diff
@@ -108,7 +125,13 @@ jobs:
           kubectl rollout status -n kube-system ds/antrea-agent --timeout=5m
       - name: Run e2e tests
         run: |
-          ./ci/run-k8s-e2e-tests.sh "--e2e-${{ inputs.test-suite }}"
+          conformance_image_options=()
+          # If the K8s version is 'latest', we need to find the latest conformance image.
+          if [ ${{ inputs.k8s-version == 'latest' }} ]; then
```

Comment:
Question：do you mean there will be corresponding conformance image on registry.k8s.io even it's rc or alpha release?

Reply:
Yes, I have checked, every Kubernetes version tag has a corresponding conformance image.